Sources/LiveViewNative/LiveViewNative.docc/Resources/01-01-06-cats_list_live.ex

```diff
@@ -24,6 +24,10 @@ defmodule LvnTutorialWeb.CatsListLive do
     {:ok, assign(socket, cats: @cats)}
   end
 
+  def render(%{platform_id: :web} = assigns) do
+    ~H""
+  end
```

Comment:
I did this to avoid adding HTML templates to the tutorial. Does this seem reasonable, or would it be better to include the HTML templates as steps in the tutorial? It may not be that important if we'll be switching to LiveBook for the tutorial anyways.

Reply:
Ok, I can update it to use the LVN sigil too.